pkg/actors/actors.go

```diff
@@ -111,7 +111,7 @@ type ActorMetadata struct {
 // ActorRemindersMetadata represents information about actor's reminders.
 type ActorRemindersMetadata struct {
 	PartitionCount int                `json:"partitionCount"`
-	partitionsEtag map[uint32]*string `json:"-"`
+	PartitionsEtag map[uint32]*string `json:"-"`
```

Comment:
This should remain unexported, as `partitionsEtag` is not being used in a package outside of `actors`.

Reply:
Done.